app/markets/Bet.tsx

```diff
@@ -60,6 +60,36 @@ export const Bet = ({ fixedProductMarketMaker }: BetProps) => {
       </div>
     );
 
+  if (marketModel.currentAnswer !== null && !marketModel.isAnswerResolved)
+    return (
+      <div className="flex flex-col items-center space-y-8 p-8">
+        <div className="flex items-center space-x-2">
+          <span>Current winner outcome is</span>
+          <Tag
+            className="w-fit uppercase"
+            colorScheme={marketModel.currentAnswer === 0 ? 'success' : 'danger'}
+          >
+            {marketModel.outcomes[marketModel.currentAnswer]?.name}
+          </Tag>
+        </div>
+        <div className="flex flex-col items-center space-y-2">
```

Comment:
this will improve the styles
```suggestion
        <div className="bg-surface-surface-1 flex flex-col items-center py-4 rounded-12 w-full">
```

Reply:
Will give it a try